generated/src/aws-cpp-sdk-dynamodb/include/aws/dynamodb/DynamoDBClient.h

```diff
@@ -2257,7 +2266,6 @@ namespace DynamoDB
       mutable Aws::Utils::ConcurrentCache<Aws::String, Aws::String> m_endpointsCache;
       DynamoDBClientConfiguration m_clientConfiguration;
       std::shared_ptr<Aws::Utils::Threading::Executor> m_executor;
```

Comment:
please don't forget to use m_executor from the config.

Reply:
yep will remove as part of next iteration with codegen